var/spack/repos/builtin/packages/fftw/package.py

```diff
@@ -102,29 +103,56 @@ def install(self, spec, prefix):
             float_options.append('--enable-sse2')
             double_options.append('--enable-sse2')
 
-        # Build double precision
-        configure(*(options + double_options))
-        make()
-        if self.run_tests:
-            make("check")
-        make("install")
+        configure = Executable('../configure')
 
-        # Build float/long double/quad variants
+        # Build double/float/long double/quad variants
+        with working_dir('double', create=True):
```

Comment:
It seems like double is the only one that isn't optional. Maybe we should make this one a variant too just for uniformity?